systemd/openqa-enqueue-asset-cleanup.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Enqueues an asset cleanup cleanup task for the openQA.
```

Comment:
Please don't copy the Czech "the" here again.

Reply:
Also, one cleanup is enough ;-)

Reply:
fixed

Reply:
there is no such thing as 'Czech "the"' - a proper classification of this construct is 'czech overcompensation of missing articles' :)

Reply:
Note that it has been copied from "the openQA database" and then only "database" has been removed.